src/main/java/io/ebeaninternal/server/core/DatabasePlatformFactory.java

```diff
@@ -134,6 +134,7 @@ private DatabasePlatform byDatabaseMeta(DatabaseMetaData metaData) throws SQLExc
     String dbProductName = metaData.getDatabaseProductName();
     dbProductName = dbProductName.toLowerCase();
 
+    // Isn't being used
     int majorVersion = metaData.getDatabaseMajorVersion();
```

Comment:
Both Eclipse and Intellij report this field is not being used from any point in the code. Leave it?